notebooks/official/tabnet/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Where is the training and serving code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
As our offline discussion, the purpose of notebook is visualization the TabNet's mask, which is used to provide explanation.

Reply:
<p>I feel it would be good to show users how to get aggregated_mask_values from TabNet trained from https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/726 in this notebook. Otherwise users don't know how to get aggregated_mask_values. Is there any feature gap on TabNet on pipeline preventing for getting aggregated_mask_values?</p>

Reply:
<p>Once above is done, we can make the TabNet on pipeline training notebook link to this one, so the user journey is more complete.</p>

Reply:
The aggregated_mask_values is part of output from estimator so there is no gap here. Not all users needs to use explanation. I have no objection to integrate two notebooks together.

Reply:
Thanks, Long. Either two separate notebooks or a single one look good to me. I slightly prefer two separate ones and make the training one link the explanation one. Due to note all users using TabNet training would need explanation. But no strong opinion.

Would it be possible for you to add an optional step for getting the aggregated_mask_values from the result of TabNet pipeline? If users don't want to run the TabNet training pipeline, we can share an existing output for visualization. Then we can figure out merge vs. keep two separate notebooks.

Reply:
Sounds good. Thank you for your sugesstion.
Since we need the notebook ready to publish the blog, I suggest to keep the content as it now.
I'll add the optional section of getting aggregated mask in the next PR.

Reply:
lgtm